mappings/net/minecraft/item/ItemUsageContext.mapping

```diff
@@ -15,6 +15,7 @@ CLASS net/minecraft/class_1838 net/minecraft/item/ItemUsageContext
 		ARG 4 stack
 		ARG 5 hit
 	METHOD method_17698 getHitPos ()Lnet/minecraft/class_243;
+	METHOD method_17699 isHitInteringBlock ()Z
```

Comment:
What is `intering`

Reply:
Whoops! Another of my typos slipped through, typing `int` when I actually mean something else...

Damn Primitives.